Makefile

```diff
@@ -38,8 +38,8 @@ wheel: setup-git setup-venv ## build a wheel of the Python library
 	cd py && ../.venv/bin/python setup.py bdist_wheel
 .PHONY: wheel
 
-wheel-manylinux: setup-git ## build manylinux 2010 compatible wheels
-	@scripts/docker-manylinux.sh
+wheel-manylinux: setup-git ## build manylinux 2014 compatible wheels, you must to have a pre-built image from py/Dockerfile
```

Comment:
This is unfortunate as we also do not have a utility (e.g. `make` command) to create the builder stage. That makes it hard(er) to reproduce this locally.

Could we move the build of that builder image into the bash script? You can still have the caches in GHA before invoking the script.

Reply:
This target now will build the image and then run the cross build inside that image